rally-domain/src/main/groovy/yakworks/rally/orgs/repo/ContactRepo.groovy

```diff
@@ -105,6 +105,14 @@ class ContactRepo extends LongIdGormRepo<Contact> {
     void afterPersist(Contact contact, AfterPersistEvent e) {
         if (contact.location?.isDirty()) contact.location.persist()
         syncChangesToUser(contact)
+        boolean isPrimary = Maps.boolean(e.data, 'isPrimary',  false)
```

Comment:
I dont think we should support this this way because of problems above. 
we should make a seperate call maybe, or need to sort out how not to call this if its coming from Org, which would be tricky